app/models/concerns/groups/licenses.rb

```diff
@@ -0,0 +1,19 @@
+module Groups::Licenses
+  extend ActiveSupport::Concern
+
+  included do
+    scope :licensed, -> { where licensed: true }
+
+    attr_accessor :users_left_count
+  end
+
+  def auditors_limit
+    licensed? ? license.auditors_limit : Rails.application.secrets[:auditors_limit]
```

Comment:
Estuve pensando y hay un _pequeño_ problema con los clientes a los que no tengo acceso. Les debería pedir, previo a la actualización el `config/application.yml`, para devolverlo con el `RAILS_MASTER_KEY`. Igual lo hacemos, es posible.

¿El `secrets.yml.enc` lo vas a subir en este PR o en otro?

Así no me pasa la gran monitor, de darle al deploy y boom... =)

Reply:
No uso ese archivo, pero te lo genero, no hay drama.

Reply:
¿Y donde lo ponés? ¿En el secret.yml pelado?

Reply:
claro, los machos usamos secrets no tan secrets jaja. Siempre uso variables, pero si te llama la paranoia lo encriptamos :+1:

Reply:
Y yo creo que ya que estamos lo hacemos así, sino le vamos a pasar la modificación del archivo que si le hacen un diff ven:

```yaml
USUARIOS: 5
```

Y nos vemos =)

Reply:
@francocatena  ahí lo agregué. Y se lo inyecté al travis, sino lo iba a leer mon toto jaja. Fijate si te parece bien así. El `secrets:edit` me dice "Está deprecado, usa credentials".

Reply:
La otra es hacer alguna de las 2 alternativas que sugiere acá:
https://medium.com/craft-academy/encrypted-credentials-in-ruby-on-rails-9db1f36d8570

Reply:
¿Está deprecado usarlo como variable de entorno?

Y si, me pintó como dice el flaco con el comando `travis` ¿Tenés permiso de hacerlo como estás en el repo? Sino decime y lo hago yo.

Reply:
No, no está deprecada la variable. Lo que se deprecó el es el `secrets:edit` ahora en vez de secrets es credentials.
Pruebo en un rato y comento.

Reply:
Ya quedo esto loco :+1:

Reply:
Buenísimo, suerte con el depurado... hermosa aventura de fin de semana =P